frontend/src/pages/team/Applications/index.vue

```diff
@@ -132,7 +132,8 @@ export default {
             columns: [
                 { label: 'Name', class: ['flex-grow'], key: 'name', sortable: true }
             ],
-            filterTerm: ''
+            filterTerm: '',
+            tour: null
```

Comment:
love the singletone approach, two questions pop into my head:
1. should we clear the tour after it's completion?
2. should the tour be in the store for a more global approach?

thinking as I write, we can move it whenever we'd need something more widespread/global

Reply:
> thinking as I write, we can move it whenever we'd need something more widespread/global

This, it's fine for now, when we have the need, then we build it